plugins/PrerenderPlugin/templates.js

```diff
@@ -92,7 +92,6 @@ const multiLocale = (mapping) => mapping && `
 	var conf = details[lang] || details[lang.substring(0, 2)];
```

Comment:
Please remove `and checksum` at line 89.

Reply:
ok